### ISSUE TITLE
Search bar please

### ISSUE THREAD

**Issue by GSP361-15575** [darktrojan/newtabtools#144]
I know it's supposed to throw the focus to the address bar, but that doesn't seem to happen reliably.

Could we get an option for a minimal search bar at the top of the page? Just something that throws a query to the default search engine?

**Comment by pintergreg** [comment]
I'd also like to see this, something like on about:home.

**Comment by ChodaBoy** [comment]
I agree. This would make for nice parity with the default home page.

**Comment by XedMada** [comment]
It would be terrible UX, but seeing as how we're having so many issues with firefox and address bar focus, it's a hack i'd appreciate while waiting for the proper solution.
